common/static/js/lang-edx.js

```diff
@@ -0,0 +1,65 @@
+var edx = edx || {},
```

Comment:
@asadiqbal08 mentioned to me this morning that he was having some difficulty getting Jasmine to work properly -- the test file was not loading some of the necessary modules after putting it in common/js directory.  I'm not sure of the specifics, however.